### PR TITLE
moving iframe-resizing logic into a plugin, fixes #21

### LINKED ISSUE #21: Each time a client connects, the iframe resizer gets created
Since the iframe resizer is initialized in the handling for the "ready" event, each time a new client connects, the iframe resizer will be re-created. This should only happen the first time a client connects.

### PR DESCRIPTION
**Pull request by dlockhart**
@rschick FYI

### CODE REVIEW

**Comment by omsmith** [comment]
Somewhat related. `Client` still has `this.lastHeight = 0;`, was that just missed when switching to iframe-resizer?

**Comment by dlockhart** [comment]
Yeah looks like it was just missed when iframe-resizer was introduced as you suggested. I'll remove it with this PR.

File: src/client.js [hunk 1]
```diff
@@ -8,8 +8,6 @@ export default class Client extends Port {
 
 		super(window.parent, '*', options);
 
-		this.lastHeight = 0;
```
Comment (dlockhart) [inline src/client.js]: Leftover artifact from the initial move to iframe-resizer -- thanks @omsmith for noticing it.

**Comment by rschick** [comment]
:+1:

**Pull request merged**